mlperf_logging/package_checker/package_checker.py

```diff
@@ -225,14 +257,14 @@ def check_systems(folder, ruleset):
             'Found too many errors in system checking, see log above for details.')
 
 
-def check_training_package(folder, ruleset, quiet, werror, rcp_bypass, rcp_bert_train_samples):
+def check_training_package(folder, usage, ruleset, quiet, werror, rcp_bypass, rcp_bert_train_samples):
     """Checks a training package for compliance.
 
     Args:
         folder: The folder for a submission package.
-        ruleset: The ruleset such as 0.6.0, 0.7.0, or 1.0.0.
+        ruleset: The ruleset such as 0.6.0, 0.7.0, 1.0.0 or hpc_1.0.0.
```

Comment:
Is hpc_1.0.0 really accepted here? I think hpc is part of usage

Reply:
You are right, I fixed this.